it/mail/chrome/messenger/appleMailImportMsgs.properties

```diff
@@ -54,11 +54,10 @@
 # Error Message
 ## @name APPLEMAILIMPORT_MAILBOX_BADPARAM
 ## @loc None
-2003=Si è verificato un errore interno, importazione non riuscita. Tentare nuovamente l'importazione.
+2003=Si è verificato un errore interno, importazione non riuscita. Riprova a importare di nuovo.
```

Comment:
CORRECTIONS NEEDED. 

"Riprova" and "di nuovo" make this sentence redundand. Please replace "Riprova" with "Prova".